BarcodeParserBuilder/Barcodes/CODE128/Code128StringParserBuilder.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using BarcodeParserBuilder.Infrastructure;
+
+namespace BarcodeParserBuilder.Barcodes.CODE128
+{
+    internal class Code128StringParserBuilder : BaseFieldParserBuilder<string?>
+    {
+        protected override string? Build(string? obj) => string.IsNullOrWhiteSpace(obj) ? null : obj;
+        protected override string? Parse(string? value) => value;
+
+        protected override bool Validate(string? value)
```

Comment:
are there certain string characters that are not allowed in a Code128 barcode?

Reply:
Not really, everything which is between 0x0-0xFF is fine for Code128. And that is what Im checking. Originally first 128 symbols were supported, but with FNC4 all up to 0xFF are supported. 
However, the FNC4 is completely transparent for the software and reader never passes it. At least the real ones I have (Zebra, Sunmi) do not even have option to ask to send them. Scanners interpret FNC4 themselves and pass to software already interpreted string.